### ISSUE TITLE
Test session verdict "fail" after certificate was awarded

### ISSUE THREAD

**Issue by smuellerDD** [usnistgov/ACVP-Server#56]
I have implemented an extension which fetches the verdict for a test session and a vsID at any arbitrary time.

After being awarded a certificate A922, I tried it on that production test session. To my surprise, the test session verdict returned false:

```
[
  {
    "acvVersion": "1.0"
  },
  {
    "passed": false,
    "results": [
      {
        "vectorSetUrl": "/acvp/v1/testSessions/3534/vectorSets/34669",
        "status": "passed"
      },
      {
        "vectorSetUrl": "/acvp/v1/testSessions/3534/vectorSets/34672",
        "status": "passed"
      }
    ]
  }
]
```

I fetched at URL `GET /acvp/v1/testSessions/3534/results HTTP/1.1`.

Environment: Prod.

**Comment by Kritner** [comment]
this is addressed release [v1.1.0.17](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.17) (which will be on prod this week) from the fixes in https://github.com/usnistgov/ACVP/issues/1144 and/or https://github.com/usnistgov/ACVP-Server/issues/63

**Comment by Kritner** [comment]
This change is on prod as of release [v1.1.0.17](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.17)
